test/README.md

```diff
@@ -46,9 +46,9 @@ For CI, there is also another subcommand to run pytest, reporting coverage
 
 ## The `pytest-pixl` plugin
 
-We provide a [`pytest` plugin](../pytest-pixl/README.md) with shared functionality for PIXL system
+We provide a [`pytest` plugin](../pytest-pixl/README.md) with shared functionality for the PIXL system
 and unit tests. This includes an `ftp_server` fixture to spin up a lightweight FTP server,
-to mock the FTP server used by the Data Safe Haven.
+to mock an FTP server being used to deposit the data.
```

Comment:
General comment based on my experience of PIXL – I have found the testing quite confusing. We have (I think):
- tests in each directory
- the ability to run all tests at once (via pytest.ini) albeit this has caveats
- the pytest-pixl plugin, which I don't really understand the motivation for why it's designed as a plugin
- integration tests – these are made slightly complicated by .env shenanigans and subtle differences with the containers in the unit tests

What's my point... I'm just trying to illustrate that there is quite a bit going on with the testing and it's scattered through various parts of the documentation.